relay-redis/src/scripts.rs

```diff
@@ -0,0 +1,60 @@
+use redis::Script;
+use std::sync::OnceLock;
+
+/// A collection of static methods to load predefined Redis scripts.
+pub struct RedisScripts;
+
+impl RedisScripts {
+    /// Loads the cardinality Redis script.
+    pub fn load_cardinality() -> &'static RedisScript {
+        static SCRIPT: OnceLock<Box<RedisScript>> = OnceLock::new();
```

Comment:
For some reason `RedisScript` is not `Sized` even though the types all seem to be `Sized`. I might be overlooking something.